pkgs/development/python-modules/jupyter_server/default.nix

```diff
@@ -82,6 +82,9 @@ buildPythonPackage rec {
   ];
 
   disabledTestPaths = [
+    # nbconvert failed: `relax_add_props` kwargs of validate has been
+    # deprecated for security reasons, and will be removed soon.
+    "tests/nbconvert/test_handlers.py"
     "tests/services/kernels/test_api.py"
     "tests/services/sessions/test_api.py"
```

Comment:
Do these fail for the same reason?

Reply:
Nope. These exclusions are even unnecessary on my machine.

Reply:
I reordered the list to make it clearer.